### PR TITLE
Update decadal page with new tab layout and content

### PR DESCRIPTION
**Pull request by geek-yang**
- Restructure decadal page with tab views "forecasts" "hindcasts" and "experiments"
- Extract content from existing EUCP website, deliverables and papers
- List link/doi to relevant datasets

### CODE REVIEW

**Comment by geek-yang** [comment]
@Peter9192 Could you please review the content on each panel? I went over the related websites, deliverables and papers, and assembled these overview texts on each panel. But I might miss something. Thanks! :stuck_out_tongue_winking_eye:

Also, I'm not so sure about the style of the list of datasets on each panel. We can renovate it a bit if necessary.

The breadcrumb feature and the brief summary of each page will be implemented in another PR, since they belong to all three pages of data catalogue.

File: content/decadal/3_experiments.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    licencse:
+    format: GRIB
+    size: ~20 TB
+    doi:
+    details: "2 experiments targeting to assess the impact of stochastic physics (SPPT) on sub-decadal time-scales. General settings: Model: IFS CY46R1; Resolution: TCo199; 1deg Ocean; Initialization: November 1981-2014; Hindcast length: 28 months; Ensemble size: 25 (first 7 months); 10 (8-28 months). Specific settings: Experiment #1: w/ SPPT; Experiment #2: w/o SPPT"
+---
+
+In EUCP project, we explore avenues to improve the next generation of climate predictions with experiments on the impact of resolution, initialisation, forecast drift reduction and short-lived climate forcings. 
+
+These experiments contribute to the [Decadal Climate Prediction Project](https://www.wcrp-climate.org/modelling-wgcm-mip-catalogue/cmip6-endorsed-mips-article/1065-modelling-cmip6-dcpp) (DCPP), component A, B and C. More details about these experiments can be found in the [experiment protocal](https://www.wcrp-climate.org/experimental-protocol) of DCPP.
```
Comment (Peter9192) [inline content/decadal/3_experiments.md:47]:
```suggestion
These experiments contribute to the [Decadal Climate Prediction Project](https://www.wcrp-climate.org/modelling-wgcm-mip-catalogue/cmip6-endorsed-mips-article/1065-modelling-cmip6-dcpp) (DCPP), component A, B and C. More details about these experiments can be found in the [experiment protocol](https://www.wcrp-climate.org/experimental-protocol) of DCPP.
```

Comment (Peter9192) [reply content/decadal/3_experiments.md:47]: Lead with this paragraph: EUCP also contributed to ... .

**Review by Peter9192 — CHANGES_REQUESTED** [review]
Hi Yang, that looks very nice already! I left a few comments.

File: content/decadal/3_experiments.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+tag: Experiments
+title: Experiments contributing to the Decadal Climate Prediction Project
+img: lead_center.png
+caption: The Lead Centre for Annual-to-Decadal Climate Prediction
+datasets:
+  - title: IPSL DCPP-A data for CMIP5 and CMIP6
```
[the remark this answers is not in the document]
Comment (geek-yang) [reply content/decadal/3_experiments.md:7]: Thanks for the comment! It is moved to the hindcast now.

File: content/decadal/3_experiments.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+tag: Experiments
+title: Experiments contributing to the Decadal Climate Prediction Project
+img: lead_center.png
```
[the remark this answers is not in the document]
Comment (geek-yang) [reply content/decadal/3_experiments.md:4]: Just grab a decadal hindcast figure from https://hadleyserver.metoffice.gov.uk/wmolc/ and a paper figure related to one experiment (https://esd.copernicus.org/articles/11/1033/2020/).

File: content/decadal/2_hindcasts.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        orcid: 0000-0002-1205-3754
+    workpackage: 1
+    licencse: CC-BY-SA
+    format: NetCDF (and maybe GRIB)
+    size: ~35 TB
+    doi:
+    details: Increased resolution decadal hindcasts with EC-Earth3 at T511/ORCA025. Initialized 1 Nov 1990-2004, 10 members each, integrated for 5 years + 2 months. Output as described in CMIP6 DCPP-A (Tier 1). 
+---
+
+Hindcasts are forecasts for the past performed with the same forecast system. They can be used to describe the climatology of the model, and thus identify systematic errors (i.e. biases) and estimate their magnitude, as well as the performance of the forecast system, from a variety of perspectives.
```
[the remark this answers is not in the document]
Comment (geek-yang) [reply content/decadal/2_hindcasts.md:56]: Revised based on your suggestion.

File: content/decadal/2_hindcasts.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Despite of the high expense, in EUCP project, sets of hindcasts are performed over a period in the past to test the forecast systems. These hindcasts contribute to the [Decadal Climate Prediction Project](https://www.wcrp-climate.org/modelling-wgcm-mip-catalogue/cmip6-endorsed-mips-article/1065-modelling-cmip6-dcpp) (DCPP). Most of these hindcasts belong to the component A of DCPP, which designs and organizes a coordinated decadal hindcast experiment, in conjunction with the seasonal prediction and climate modelling communities, and the production of a comprehensive archive of results for research and applications.
```
[the remark this answers is not in the document]
Comment (geek-yang) [reply content/decadal/2_hindcasts.md:58]: Corrected!

File: content/decadal/1_forecasts.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+In EUCP project, all the latest information about existing global climate prediction systems is collected and evaluated to identify the main factors that could improve their societal impact. These pre-existing climate predictions from leading organisations around the world, help us identify the relative merits of decadal predictions from different sources, and provide guidance on the use of existing climate predictions.
+
+Datasets listed below are provided by [The Lead Centre for Annual-to-Decadal Climate Prediction](https://hadleyserver.metoffice.gov.uk/wmolc/).
```
[the remark this answers is not in the document]
Comment (geek-yang) [reply content/decadal/1_forecasts.md:23]: Corrected!

File: content/decadal/1_forecasts.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+In EUCP project, all the latest information about existing global climate prediction systems is collected and evaluated to identify the main factors that could improve their societal impact. These pre-existing climate predictions from leading organisations around the world, help us identify the relative merits of decadal predictions from different sources, and provide guidance on the use of existing climate predictions.
```
[the remark this answers is not in the document]
Comment (geek-yang) [reply content/decadal/1_forecasts.md:21]: Corrected!

File: content/decadal/3_experiments.md (hunk 1 above)
Comment (geek-yang) [reply content/decadal/3_experiments.md:34]: Corrected!

File: content/decadal/1_forecasts.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+---
+tag: Forecasts
+title: Decadal climate predictions in EUCP
+img: lead_center.png
+caption: The Lead Centre for Annual-to-Decadal Climate Prediction
```
Comment (Peter9192) [inline content/decadal/1_forecasts.md:5]: This image *is* not the Lead Centre, is it? And perhaps link to the source.

File: content/decadal/1_forecasts.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+Decadal climate predictions forecast the future evolution of climate for timescales ranging from a month to a decade into the future. The main characteristic of these predictions is that they require accurate information about the present (or ‘initial’) state of the climate system since it provides some predictability for the future climate. They are therefore often termed ‘initialised’ simulations. Decadal climate predictions come with their challenges: they require sets of hindcasts that are very expensive to run (these test the model by running it over a period in the past and comparing the simulations to observations), strategies to replicate observational uncertainty, and they strongly rely on a consistent observation system. However, improving decadal climate forecasting is crucial, since it covers vital timescales for decision makers, such as climate predictions for the next few years ahead.
+
+Please visit the website of the [The Lead Centre for Annual-to-Decadal Climate Prediction](https://hadleyserver.metoffice.gov.uk/wmolc/) for an up to date overview of the latest climate prediction data.
```
Comment (Peter9192) [inline content/decadal/1_forecasts.md:22]: In this case, the "list of datasets" header below doesn't really make sense. We could remove it altogether, or put this last sentence under a renamed "Data access" header.

File: content/decadal/3_experiments.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
+---
+tag: Experiments
+title: Experiments contributing to the Decadal Climate Prediction Project
+img: calibration.png
+caption: An example showing the steps of the full calibration methods and calibration of the dynamically decomposed variables (C.H.O’Reilly et al.)
```
Comment (Peter9192) [inline content/decadal/3_experiments.md:5]: Can the caption source be a link (or at least a DOI so I can find it)?

File: content/decadal/2_hindcasts.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
+---
+tag: Hindcasts
+title: Decadal hindcasts in EUCP
+img: decadal_hindcast.png
+caption: Timeseries for Global Mean Near Surface Temperature
```
Comment (Peter9192) [inline content/decadal/2_hindcasts.md:5]: Please include the source.

File: pages/decadal.vue [hunk 12]
```diff
@@ -1,36 +1,69 @@
...
+      </button>
+    </div>
+    <div class="flex gap-8 m-8">
+      <div class="flex flex-col gap-4 min-w-2/3">
+        <h1 class="text-xl">
+          {{ tab.title }}
+        </h1>
+        <nuxt-content class="break-words prose" :document="tab" />
+        <h1 class="text-xl">
+          List of datasets
```
Comment (Peter9192) [inline pages/decadal.vue:21]:
```suggestion
          Data access
```

File: pages/decadal.vue [hunk 13]
```diff
@@ -1,36 +1,69 @@
...
+        <ul
+          v-for="(dataset, i) in tab.datasets"
+          :key="i"
+        >
+          <li>{{ dataset.title }}</li>
+          <li>Contact: {{ dataset.contact[0].name }}</li>
+          <li>Data access:</li>
+          <li class="break-words italic underline">
+            <a :href="dataset.doi" target="blank">{{ dataset.doi }}</a>
+          </li>
```
Comment (Peter9192) [inline pages/decadal.vue:32]:
```suggestion
          <li class="prose">DOI: <a :href="dataset.doi" target="blank">{{ dataset.doi }}</a></li>
```
`prose` class also does link formatting, and makes it consistent with the rest of the layout.

Perhaps we can exclude the "https://www.doi/org/" part of the displayed doi (only include it in the link target).

**Review by Peter9192 — APPROVED** [review]
Much better! I still have some suggestions. Feel free to merge when addressed.

File: content/decadal/2_hindcasts.md (hunk 11 above)
Comment (geek-yang) [reply content/decadal/2_hindcasts.md:5]: The source link is added to the caption.

File: content/decadal/1_forecasts.md (hunk 8 above)
Comment (geek-yang) [reply content/decadal/1_forecasts.md:5]: Thanks for spotting this. Forgot to replace it. Now it is the correct figure, with the link to the source.

File: content/decadal/3_experiments.md (hunk 10 above)
Comment (geek-yang) [reply content/decadal/3_experiments.md:5]: I put the doi there now.

File: content/decadal/1_forecasts.md (hunk 9 above)
Comment (geek-yang) [reply content/decadal/1_forecasts.md:22]: Good point! Let's use "Data access".

File: pages/decadal.vue (hunk 13 above)
Comment (geek-yang) [reply pages/decadal.vue:32]: Corrected! And the "https://www.doi/org/" is not displayed.

**Pull request merged**